client/channel.go

```diff
@@ -1,13 +1,14 @@
 package client
 
 import (
-	"errors"
+	"fmt"
+	"github.com/pkg/errors"
 
 	channels "github.com/replicatedhq/replicated/gen/go/v1"
 	"github.com/replicatedhq/replicated/pkg/types"
 )
 
-func (c *Client) ListChannels(appID string, appType string) ([]types.Channel, error) {
+func (c *Client) ListChannels(appID string, appType string, appSlug string) ([]types.Channel, error) {
```

Comment:
you're gonna start seeing appSlug passed in everywhere because the vendor/v3 endpoint uses it to list channels.